ProcessMaker/Http/Controllers/Api/UserTokenController.php

```diff
@@ -121,6 +216,34 @@ public function show(Request $request, User $user, $tokenId)
      * @param  \Illuminate\Http\Request  $request
      * @param  string  $tokenId
      * @return \Illuminate\Http\Response
+     * @OA\Delete(
+     *     path="users/{user_id}/tokens/{token_id}",
```

Comment:
@AbdelilahLbardi  missing the `/` in the beginning of the path should be
`/users/{user_id}/tokens/{token_id}`